great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -210,6 +212,24 @@ def metrics_parameter_builders_by_domain_type(
         """
         pass
 
+    @property
+    @abstractmethod
+    def variables(self) -> Optional[Dict[str, Any]]:
+        """
+        Returns:
+            Optional "variables" configuration attribute name/value pairs (overrides), commonly-used in Builder objects.
+        """
+        pass
+
+    @property
+    @abstractmethod
+    def rules(self) -> Optional[Dict[str, Dict[str, Any]]]:
```

Comment:
is there a better type hint we can include here?

Reply:
@donaldheppner Not currently, because it has to be exact as what we have in the `BaseRuleBasedProfiler.run()` method.